conf/flavors.yaml.template

```diff
@@ -0,0 +1,9 @@
+FLAVORS:
+  # 6 CPU, 24 GB RAM, 100 GB disk
+  DEFAULT: satqe-ssd.standard.std
+  # 16 CPU, 32 GB RAM, 160 GB disk
```

Comment:
I think we can keep this yaml as is (without .template) similar to supportability.yaml, wdyt @jyejare @JacobCallahan  ?